README.md

```diff
@@ -8,44 +8,177 @@ We need a [vimo-server](https://github.com/VCG/vimo-server) to use the [vimo-ske
 `npm install @vimo-public/vimo-sketches`
 
 ### Usage
+
+#### General Case
 ```javascript
 import { Sketch } from '@vimo-public/vimo-sketches';
 ```
 
 ```javascript
-<Sketch vimo_server={vimo_server} data_server={data_server} data_version={data_version} token={token} isQuerying={isQuerying} processRequest={processRequest}/>
+<Sketch processRequest={processRequest}/>
+```
+
+If you want to add constraint fields to node or edge, please refer react-awesome-query-builder [config fields](https://github.com/ukrbublik/react-awesome-query-builder/blob/master/CONFIG.adoc#configfields).
+
+````javascript
+const attributes = {
+  NodeFields: {
```

Comment:
What if I want to specify multiple node properties? Do I define an array? Can you extend the example to include two node properties, please?